code/game/objects/items/storage/belt.dm

```diff
@@ -160,7 +160,7 @@
 		"/obj/item/reagent_container/syringe",
 		"/obj/item/storage/pill_bottle",
 		"/obj/item/clothing/gloves/latex",
-		"/obj/item/reagent_container/hypospray/autoinjector",
+		"/obj/item/reagent_container/hypospray",
```

Comment:
The other belt can hold this already.

Reply:
What's your point? No one uses the hypospray, and this is a big part of the reason.

The rig will never really be viable as is because it just isn't anywhere close to as space efficient when it comes to the essentials a medic needs to carry.

Reply:
One is for carrying a large number of supplies, the other for carrying different things. I favor the rig, personally.

Reply:
A minority of people use backpacks as a medic; that doesn't make it good or actually competitive with the satchel.

Reply:
There's also armor, webbing and pouches to hold it as alternatives.
If you want more supplies, carry the belt. If you want things to be more handy, carry the rig.

Reply:
The function of the hypospray is not to make your pet choice more viable or palatable, nor will it do that, even if you get what you want, which is anti-QoL for no particularly good/compelling reason.